log/zap/logger.go

```diff
@@ -41,5 +41,5 @@ func (l *Logger) Error(msg string) {
 
 // Infof logs a message at info priority
 func (l *Logger) Infof(msg string, args ...interface{}) {
-	l.logger.Infof(msg, args)
+	l.logger.Infof(msg, args...)
```

Comment:
kinda hard to testing logging without some kind of local backend which zap doesn't provide. I can write one and plug it into zap but seems a bit overkill